config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Google LLC
```

Comment:
I'm not reviewing this until the other PRs go through, then we'll handle the chages.